fmriprep/workflows/bold/confounds.py

```diff
@@ -9,23 +9,20 @@
 .. autofunction:: init_ica_aroma_wf
 
 """
-from niworkflows.nipype.pipeline import engine as pe
-from niworkflows.nipype.interfaces import utility as niu, fsl
-from niworkflows.nipype.interfaces.nilearn import SignalExtraction
-from niworkflows.nipype.algorithms import confounds as nac
-
-from niworkflows.interfaces.segmentation import ICA_AROMARPT
+from niworkflows.interfaces.fixes import \
+    FixHeaderApplyTransforms as ApplyTransforms
 from niworkflows.interfaces.masks import ROIsPlot
-from niworkflows.interfaces.fixes import FixHeaderApplyTransforms as ApplyTransforms
+from niworkflows.interfaces.segmentation import ICA_AROMARPT
+from niworkflows.nipype.algorithms import confounds as nac
+from niworkflows.nipype.interfaces import fsl
+from niworkflows.nipype.interfaces import utility as niu
+from niworkflows.nipype.interfaces.nilearn import SignalExtraction
+from niworkflows.nipype.pipeline import engine as pe
 
-from ...interfaces import (
-    TPM2ROI, AddTPMs, AddTSVHeader, GatherConfounds, ICAConfounds,
-    DerivativesDataSink
-)
-from ...interfaces.patches import (
-    RobustACompCor as ACompCor,
-    RobustTCompCor as TCompCor
-)
+from ...interfaces import (TPM2ROI, AddTPMs, AddTSVHeader, DerivativesDataSink,
+                           GatherConfounds, ICAConfounds)
+from ...interfaces.patches import RobustACompCor as ACompCor
+from ...interfaces.patches import RobustTCompCor as TCompCor
```

Comment:
And again.

Reply:
Sorry, [isort](https://pypi.org/project/isort/) on save sorted the imports.